src/plugins/lua/known_senders.lua

```diff
@@ -72,6 +77,23 @@ local function make_key(input)
   return hash:hex()
 end
 
+local function make_key_replies(goop, sz, prefix)
+  local h = rspamd_cryptobox_hash.create()
+  h:update(goop)
+  local key
+  if sz then
+    key = h:base32():sub(1, sz)
+  else
+    key = h:base32()
+  end
```

Comment:
```suggestion
  local key = prefix or '' .. h:base32():sub(1, sz)
```

As `string.sub('hui', 1, nil) == 'hui'`